### PR TITLE
OCPBUGS-10810: oc-mirror example in agent docs

### PR DESCRIPTION
**Pull request by skopacz1**
[OCPBUGS-10810](https://issues.redhat.com/browse/OCPBUGS-10810) and [OSDOCS-5962](https://issues.redhat.com/browse/OSDOCS-5962)

This PR adds an oc-mirror example to the Agent Installer doc for using the Agent with a mirror registry. It also restructures the instructional content of the page into a procedure as part of [OSDOCS-5962](https://issues.redhat.com//browse/OSDOCS-5962).

Versions: 4.12+

QE review:
- [x] QE has approved this change.

Link to current doc: [About mirroring the OpenShift Container Platform image repository for a disconnected registry](https://docs.openshift.com/container-platform/4.12/installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.html#agent-install-about-mirroring-for-disconnected-registry_understanding-disconnected-installation-mirroring)

Docs preview: 
https://59481--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.html#agent-install-about-mirroring-for-disconnected-registry_understanding-disconnected-installation-mirroring

### CODE REVIEW

File: modules/agent-install-configuring-for-disconnected-registry.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+[source,yaml]
+----
+  additionalTrustBundle: |
+    -----BEGIN CERTIFICATE-----
+    ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+    -----END CERTIFICATE-----
+----
+
+. If you are using {ztp} manifests: add the `registries.conf` and `ca- bundle.crt` files  to the `mirror/directory` path to add the mirror configuration in the agent ISO image.
```
Comment (bfournie) [inline modules/agent-install-configuring-for-disconnected-registry.adoc:42]: Should not have space i.e. 'ca-bundle.crt'

Comment (bfournie) [inline modules/agent-install-configuring-for-disconnected-registry.adoc:42]: The name of the mirror directory is just 'mirror'

File: modules/agent-install-configuring-for-disconnected-registry.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+[source,toml]
+----
+[[registry]]
+location = "registry.ci.openshift.org/ocp/release" mirror-by-digest-only = true
+
+[[registry.mirror]] location = "virthost.ostest.test.metalkube.org:5000/localimages/local-release-image"
+
+[[registry]]
+location = "quay.io/openshift-release-dev/ocp-v4.0- art-dev" mirror-by-digest-only = true
```
Comment (zniu1011) [inline modules/agent-install-configuring-for-disconnected-registry.adoc:59]: Shouldn't have apace in 'ocp-v4.0- art-dev'

Comment (zniu1011) [inline modules/agent-install-configuring-for-disconnected-registry.adoc:59]: Shouldn't have apace in 'ocp-v4.0- art-dev'

Comment (skopacz1) [reply modules/agent-install-configuring-for-disconnected-registry.adoc]: fixed, thanks for catching this! Let me know if you have any other feedback

File: modules/agent-install-configuring-for-disconnected-registry.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+You must use the output of either the `oc adm release mirror` command or the oc-mirror plugin to configure the Agent-based Installer to use mirrored images.
+
+.Procedure
+
+. If you used the oc-mirror plugin to mirror your release images:
+
+.. Open the `imageContentSourcePolicy.yaml` located in the results directory, for example `oc-mirror-workspace/results-1682697932/`.
+
+.. Copy the text in t he `repositoryDigestMirrors` section of the yaml file.
```
Comment (GroceryBoyJr) [inline modules/agent-install-configuring-for-disconnected-registry.adoc:16]:
```suggestion
.. Copy the text in the `repositoryDigestMirrors` section of the YAML file.
```

**Review by GroceryBoyJr — COMMENTED** [review]
@skopacz1 only a few small nits sir. Nice work!

File: modules/agent-install-configuring-for-disconnected-registry.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc
+
+:_content-type: Procedure[id="agent-install-configuring-for-disconnected-registry_{context}"]
+= Configuring the Agent-based Installer to use mirrored images
+
+You must use the output of either the `oc adm release mirror` command or the oc-mirror plugin to configure the Agent-based Installer to use mirrored images.
```
[the remark this answers is not in the document]
Comment (skopacz1) [reply modules/agent-install-configuring-for-disconnected-registry.adoc:8]: Good catch, thanks! Though I've been trying to consistently refer to it as a plugin, so it's the later statement that I need to fix.

**Pull request merged**